src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/tempListView.selector.js

```diff
@@ -8,10 +8,10 @@ import {
     Option,
     DataElement,
     dataElementTypes,
-} from '../../../metaData';
-import getStageFromEvent from '../../../metaData/helpers/getStageFromEvent';
-import { convertValue } from '../../../converters/clientToList';
-import RenderFoundation from '../../../metaData/RenderFoundation/RenderFoundation';
+} from '../../../../metaData';
+import getStageFromEvent from '../../../../metaData/helpers/getStageFromEvent';
+import { convertValue } from '../../../../converters/clientToList';
+import RenderFoundation from '../../../../metaData/RenderFoundation/RenderFoundation';
```

Comment:
I have mentioned this the #1 of this chain. Maybe we could change it. Lgmt doesnt like it much and it seems to me that we can simply add the `type` keyword. Not a blocker of course 
```suggestion
import type { RenderFoundation } from '../../../../metaData';
```

Reply:
This file is completely removed later.